### PR TITLE
notify-send: set app-name and urgency

### PR DESCRIPTION
**Pull request by joehillen**

### CODE REVIEW

**Comment by aristidesfl** [comment]
Can you please include a screenshot of what this looks like? Thank you

**Comment by joehillen** [comment]
I use dunst, so my setup is custom, but here you go: image

**Comment by aristidesfl** [comment]
And what does it look like without app name?

File: conf.d/done.fish [hunk 1]
```diff
@@ -69,7 +69,11 @@ and test -n __done_get_window_id  # is able to get window id
 				osascript -e "display notification \"$message\" with title \"$title\""
 
 			else if type -q notify-send # Linux notify-send
-				notify-send --icon=terminal "$title" "$message"
+				set -l crit
```
Comment (aristidesfl) [inline conf.d/done.fish]: No need to set empty variable

File: conf.d/done.fish [hunk 2]
```diff
@@ -69,7 +69,11 @@ and test -n __done_get_window_id  # is able to get window id
...
+				if test $exit_status -ne 0
+					set crit "--urgency=critical"
```
Comment (aristidesfl) [inline conf.d/done.fish]: `$urgency` would be easier to read

**Comment by joehillen** [comment]
It's a 2 line patch. Do whatever you want

**Pull request merged**